File: ej3extra/src/ej3extra/Ej3extra.java
package ej3extra;

import javax.swing.JOptionPane;

class Ej3extra {

	public static void main(String[] args) {
		double radio, perim, area;
		final double p=3.1416;
		
		radio=Double.parseDouble(JOptionPane.showInputDialog("Ingrese radio"));
		perim=2*p*radio;
		area=p*radio*radio;
		JOptionPane.showMessageDialog(null, "perimentro: " + perim + "\narea: " + area);
		

	}

}

File: Ej3extra.js
let radio;
let perim;
let area;

const p = 3.1416;


radio = parseFloat(prompt("Ingrese radio: "));
perim = 2 * p * radio;
area = p * (radio * radio);

alert("El perimentro es: " + perim + "\nEl area es: " + area);
